roles/kubernetes-apps/ansible/templates/kubedns-deploy.yml

```diff
@@ -83,6 +83,9 @@ spec:
 {% if kube_log_level == '4' %}
         - --log-queries
 {% endif %}
+{% if dns_mode == 'kubedns' %}
```

Comment:
I think there's little value in making this a conditional. If you want to merge as-is, sure, but I think it's okay to block bogus domains always.